vms/avm/genesis_builder_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package avm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/formatting"
+	"github.com/ava-labs/avalanchego/utils/formatting/address"
+)
+
+var addrStrArray = []string{
```

Comment:
nit: move this to inside `TestBuildGenesis`. Shared information across tests tend to add more confusion imo.